packages/lesswrong/components/posts/PostSubmit.tsx

```diff
@@ -106,7 +106,7 @@ const PostSubmit = ({
       <div className={classes.submitButtons}>
         {requestFeedbackKarmaLevel !== null && currentUser.karma >= requestFeedbackKarmaLevel && document.draft!==false && <LWTooltip
           // EA Forum title is Effective Altruism Forum, which is unecessarily long
-          title={`Request feedback from the ${isEAForum ? "EA Forum" : forumTitleSetting.get()} team.`}
+          title={isEAForum ? 'Request feedback from the EA Forum team' : 'Request feedback from our editor'}
```

Comment:
I'd recommend still leaving `forumTitleSetting.get()` as the default here and making the new copy forum-gated on `isLWorAF`. The EA forum and LW/AF are not the only sites using ForumMagnum. Otherwise, no objections from EAF.